ProjFS.Mac/PrjFSKext/public/PrjFSCommon.h

```diff
@@ -10,7 +10,7 @@
 #define PrjFSServiceClass       "org_vfsforgit_PrjFS"
 
 // TODO: move this to an autogenerated header. (#648)
-#define PrjFSKextVersion "0.1.1"
+#define PrjFSKextVersion "0.1.2"
```

Comment:
What circumstances do we up this number?

Reply:
When the kext/userlib interface api changes. (In this case, message format.)